.vsts-ci/templates/publish-markets.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+- checkout: self
+
+- download: current
+  artifact: vscode-powershell
+  displayName: Download signed artifacts
+
+- pwsh: |
+    vsce publish --packagePath $(Pipeline.Workspace)/powershell-*.vsix --pat $(VsceToken)
```

Comment:
I think there might be existing tasks for this, like [this one](https://marketplace.visualstudio.com/items?itemName=ms-devlabs.vsts-developer-tools-build-tasks).

Just a suggestion in case you'd like to reuse an existing solution. We could always try moving to it later on too.

Reply:
Thanks! I didn't even think to look for a task for this. At a glance that particular task looks like it's for the _Visual Studio_ marketplace (not VS Code) but maybe it would work anyway? To be frank, I haven't been able to test this yet because, well, it'll do a release if I test it.

Also just realizing I don't install the `vsce` tool so this isn't going to work...

Reply:
Ok installed `vsce` (probably).

Reply:
Are they the same marketplace? 🤔